tests/integration/test_api_resources.py

```diff
@@ -6,7 +6,13 @@
 from datetime import datetime
 import time
 
-import mock
+import six
+
+if six.PY3:
+    from unittest import mock
+else:
+    import mock
+
```

Comment:
Maybe something like `from tests.utils import mock`, and move repeating logic there

Reply:
nice, will do